docs/website/docs/developers/general/developer-tips.md

```diff
@@ -157,7 +157,8 @@ Flag | Files dumped
     $ mkdir -p /tmp/iree/simple_abs/
 
     $ iree-compile simple_abs.mlir \
-      --iree-hal-target-backends=llvm-cpu \
+      --iree-hal-target-device=local \
+      --iree-hal-local-target-device-backends=llvm-cpu \
```

Comment:
Change hl_lines from `5 6` to `6 7` here (just outside the range where I can make a suggested edit, thanks github)